src/jobs/models/kubernetes/jobExperimentsHandler.js

```diff
@@ -77,5 +81,6 @@ function convertDurationStringToMillisecond(durationString) {
 module.exports = {
     setChaosExperimentsIfExist,
     scheduleChaosExperiment,
-    stopChaosExperimentsForJob
+    stopChaosExperimentsForJob,
+    clearAllFinishedJobExperiments
```

Comment:
why is it exposed externally - is it only to allow unit tests stubs

Reply:
no, it is called by k8s jobConnector